lib/modable.rb

```diff
@@ -79,13 +79,34 @@ def winningest_coach1(id)
     games_won_hash = games_won.group_by{ |game| game.head_coach}
     numb_games_won = games_won_hash.each{ |k,v| games_won_hash[k] = v.length}
     numbers = []
-    result = {}
+    @result = {}
     numb_games_won.each{ |k,v| games_played.each{ |k1,v1|
        if k == k1
-          result[k] = (v.to_f/v1.to_f).round(4)
+          @result[k] = (v.to_f/v1.to_f).round(4)
+       end
+       }}
+  end
+
+  def worst_coach1(season)
```

Comment:
When we do our final merge we should go back and make sure we didn't duplicate methods. Some of these look really familiar! lol